install.sh

```diff
@@ -163,8 +163,8 @@ if [ "$DISTRIBUTION" = "Darwin" ]; then
   OS="macOS"
 elif [ -f /etc/debian_version ] || [ "$DISTRIBUTION" == "Debian" ] || [ "$DISTRIBUTION" == "Ubuntu" ]; then
   OS="Debian"
-elif [ "$AWS_EXECUTION_ENV" == "CloudShell" ]; then
-  OS="AWSCloudShell"
+elif [ "$AWS_EXECUTION_ENV" == "CloudShell" ] || [ "$POWERSHELL_DISTRIBUTION_CHANNEL" == "CloudShell" ]; then
+  OS="CloudShell"
```

Comment:
How do we distinguish between the different cloud's cloudshell then?

Reply:
LGTM in terms of fixing the issue. 

@chris-rock - do we currently have a use case where we want to differentiate between various cloud shells?
If yes, we could change the match condition to something like `[[ "$OS" = *CloudShell ]]` and prefix the different shells with the provider?

Reply:
Not right now, but in future iterations we want to make sure the script runs in AWS, Azure and GCP cloudshells.